### PR TITLE
fix: linting

### PR DESCRIPTION
**Pull request by capaj**
- runs lint on CI now
- fixes lint command to actually check the project

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
   "parser": "@typescript-eslint/parser",
-  "extends": ["standard", "prettier"],
+  "extends": [
+    "standard",
+    "prettier",
```
Comment (simoneb) [inline .eslintrc:5]: standard and prettier should really not be used together

**Review by simoneb — COMMENTED** [review]
One comment, also build is failing so this PR may need more changes

File: .eslintrc (hunk 1 above)
Comment (capaj) [reply .eslintrc:5]: they can be used together-formatting rules from the first get overriden by the second, so it should not be a technical issue. Standard has a few nice linting rules for writing node.js callback style code for example.
I will remove it if you think it's not worth it.

Comment (capaj) [reply .eslintrc:5]: This would remove import order eslint rules as well-are you sure we don't want to keep those?

Comment (simoneb) [reply .eslintrc:5]: if they don't collide let's keep them, I didn't know they could live together :)

**Review by simoneb — APPROVED** [review]

**Pull request merged**